shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -151,23 +159,9 @@ case class Arrival(Operator: Option[Operator],
     )
 
     preferredSources
-      .find { case (source, _) => TotalPax.get(source).exists(_.isDefined) }
-      .map { case (source, fn) => TotalPaxSource(fn(TotalPax(source)), source) }
-      .orElse(fallBackToFeedSource)
-      .getOrElse(TotalPaxSource(None, UnknownFeedSource))
-  }
-
-  def fallBackToFeedSource: Option[TotalPaxSource] = {
```

Comment:
Now this function is being removed, we need to deal with deserialising an `Arrival` with no pax sources and only actpax and transpax - also really need a test around that scenario